src/python/dxpy/templating/templates/nextflow/resources/usr/local/bin/dx-registry-login

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+CREDENTIALS=credentials
+if [[ ! -f "$CREDENTIALS" ]]; then
+    exit 0 # No credentials - we carry on without them.
+fi
+
+set -e
+
+
+if [[ $1 == "--help" || $1 == "-h" ]]; then
+    echo -e "Usage: $0"
+    echo -e "Logins into a Docker registry specified in credentials input file"
+    exit
+fi
+
+command -v docker >/dev/null 2>&1 || (echo "ERROR: docker is required when running with the Docker credentials."; exit 1)
+
+export REGISTRY=$(jq '.docker_registry.registry' "$CREDENTIALS" | tr -d '"')
+export REGISTRY_USERNAME=$(jq '.docker_registry.username' "$CREDENTIALS" | tr -d '"')
+export REGISTRY_ORGANIZATION=$(jq '.docker_registry.organization' "$CREDENTIALS" | tr -d '"')
+if [[  -z $REGISTRY_ORGANIZATION || $REGISTRY_ORGANIZATION == "null" ]]; then
+    export REGISTRY_ORGANIZATION=$REGISTRY_USERNAME
+fi
+
+if [[ -z $REGISTRY || $REGISTRY == "null" \
+      || -z $REGISTRY_USERNAME  || $REGISTRY_USERNAME == "null" ]]; then
+    echo "Error parsing the credentials file. The expected format to specify a Docker registry is: "
+    echo "{"
+    echo "    docker_registry: {"
```

Comment:
I forgot if Nextflow has its own convention for providing Docker info? If so IMHO we should follow it.

Reply:
You can merge it as is but if needed, please create a JIRA ticket for this. Thanks